frontend/src/components/OrganizationSearch.vue

```diff
@@ -0,0 +1,166 @@
+<!--suppress XmlInvalidId -->
+<template>
+  <div>
+    <!-- Basic Search -->
+    <v-row no-gutters>
+      <v-col class="col-6">
+        <label for="organization-search" >
+          Search
+          <v-tooltip right>
+            <template v-slot:activator="{ on }">
+              <v-icon v-on="on" small>mdi-help-circle</v-icon>
+            </template>
+            <span>Search by organization name or ID</span>
+          </v-tooltip>
+        </label>
+
+        <v-text-field
+          id="organization-search"
+          outlined
+          dense
+          v-model="organizationSearchInput"
+          placeholder="Organization or ID"    
+          @keyup.enter="searchOrganization('&search='+organizationSearchInput.replaceAll('\\','%5C'))"     
+        />
+      </v-col>
+      <v-col class="col-4">
+          <v-btn id="search-button" class="primary" medium @click.native="searchOrganization('&search='+organizationSearchInput.replaceAll('\\','%5C'))">Search Organizations</v-btn>
+      </v-col>
+      <v-col class="col-2">
+        <v-btn id="create-organization-button" class="success" medium @click.native="goToCreateOrganization">Create New organization</v-btn>
+      </v-col>
+    </v-row>
+
+    <!-- table -->
+    <v-row no-gutters>
+      <v-col class="col-12">
+        <v-data-table
+          id="organizations-table"
+          class="base-table select-table"
+          :search = "organizationSearchInput"
+          :headers="headers"
+          :items="searchResults"
+          :footer-props="footerProps"
+          :loading="organizationSearchLoadingStatus"
+          loading-text="Searching for organizations"
+          v-on:click:row="selectOrganization"
+        >
+          <!-- https://stackoverflow.com/questions/61394522/add-hyperlink-in-v-data-table-vuetify -->
+          <template #item.id="{ item }">
+            <a target="_blank" :href="`#/organizations/${item.id}`" v-on:click="openNewTab">
+              {{ item.id }}
+            </a>
+            <v-icon small>mdi-open-in-new</v-icon>
+          </template>
+          <template v-slot:footer>
+            <v-toolbar flat>
+              <v-spacer/>
+              <download-csv
+                  :data="searchResults"
+                  :fields="['id','name']"
+              >
+                <v-btn id="csv-button" class="primary" small>Download results</v-btn>
```

Comment:
Downloading results is not needed at this time.

Reply:
removed download result button